clients/admin-ui/src/constants.ts

```diff
@@ -1,51 +1,9 @@
-export interface User {
-  id?: string;
-  first_name?: string;
-  last_name?: string;
-  username?: string;
-  password?: string;
-  created_at?: string;
-}
+import { UserPrivileges } from './features/user-management/types';
 
-export interface UserResponse {
-  id: string;
-}
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I think there are >1 exports now, so don't need this line anymore?

Reply:
Yup yup yup!